tool/behave_rule.bzl

```diff
@@ -83,9 +83,9 @@ def _rule_implementation(ctx):
              ./typedb_distribution/"$DIRECTORY"/typedb server --server.address 0.0.0.0:$PORT --storage.data typedb_test &
             else
             ./typedb_distribution/"$DIRECTORY"/typedb cluster \
-                --server.address=127.0.0.1:$PORT \
-                --server.internal-address.zeromq=127.0.0.1:$(($PORT+1)) \
-                --server.internal-address.grpc=127.0.0.1:$(($PORT+2)) \
+                --server.address=localhost:$PORT \
```

Comment:
I've added an asana task for Dmitrii U to try to upgrade `tonic` so we can use either `localhost` or `127.0.0.1`, since having that option is pretty nice